rancher/charts
# Change the monitoring coredns network policy to an "allow all" policy.

## Issue: https://github.com/rancher/rancher/issues/42809

## Problem
kubectl get all (or any kubectl command) outputs the following error:
Error from server (InternalError): an error on the server ("dial tcp: lookup kubernetes. Default on 10.43.0.10:53: read udp 10.42.0.26:56849->10.43.0.10:53: i/o timeout") has prevented the request from succeeding

This bug also prevents the monitoring chart from being deleted as the Kubernetes API calls hang. 

## Solution
Change the monitoring-network-policy to a default allow all policy, to allow proper network traffic. 

## Testing
Install monitoring chart version 103.0.0+up40.1.2 on rancher version 2.8 and execute any kubectl command from the Rancher UI. 

## Engineering Testing
### Manual Testing
Installed monitoring chart version 103.0.0+up40.1.2 on rancher version 2.8 and executed any `kubectl get all` from the Rancher UI kubectl shell and received the correct response. Successfully deleted, reinstalled, and upgraded monitoring chart. 

As this change touches the same monitoring policy for https://github.com/rancher/rancher/issues/38820, I tested this change in a hardened cluster to ensure that no targets are down. 

### Automated Testing
N/A

## QA Testing Considerations
Fresh install, upgrade, deletion of monitoring chart. 



### Regressions Considerations
N/A 

## Backporting considerations
N/A as bug was introduced by the fix for https://github.com/rancher/rancher/issues/38820 in this PR: https://github.com/rancher/charts/pull/2868.
